core/modules/server/server-sent-events.js

```diff
@@ -0,0 +1,65 @@
+/*\
+title: $:/core/modules/server/server-sent-events.js
+type: application/javascript
+module-type: library
+\*/
+(function(){
+
+/*jslint node: true, browser: true */
+/*global $tw: false */
+"use strict";
+
+/*
+parameters:
+		prefix - usually the plugin path, such as `plugins/tiddlywiki/tiddlyweb`. The
+			route will match `/events/${prefix}` exactly.
+
+		handler - a function that will be called each time a request comes in with the
+			request and state from the route and an emit function to call.
+*/
+
+var ServerSentEvents = function ServerSentEvents(prefix, handler) {
+  this.handler = handler;
```

Comment:
We use tabs with a width of 4 for indenting, it looks like these files are indented with spaces.

Reply:
Fixed.